rules/apps.libsonnet

```
@@ -205,6 +205,28 @@
       {
         name: 'k8s.rules.pod_owner',
         rules: [
+          // workload aggregation for replicasets
+          {
+            record: 'namespace_workload_pod:kube_pod_owner:relabel',
+            expr: |||
+              max by (%(clusterLabel)s, namespace, workload, pod, workload_type) (
```

`workload_type` label is added by line 227 and doesn't exist in this query

```suggestion
              max by (%(clusterLabel)s, namespace, workload, pod) (
```